RELEASE.md

```diff
@@ -0,0 +1,61 @@
+# Releases
+
+## Skada v0.4.0
+
+This update brings significant enhancements and new features:
+1. New Shallow Methods: MongeAlignment and JCPOT
+2. New Deep Methods: CAN, MCC, MDD, SPA, SourceOnly, and TargetOnly models.
+3. Scorers: Introduced MixValScorer and improved scorer compatibility with deep models.
+4. Subsampling Transformers: Added StratifiedDomainSubsampler and DomainSubsampler.
+5. Deep Models: Enhanced batch handling, fixed predict_proba, stabilized MDD loss, and fixed Deep Coral.
+6. Docs & Design: Added a contributor guide, new logo, and documentation updates.
+
+### What's Changed
+* Update README.md with zenodo badge by @rflamary in https://github.com/scikit-adaptation/skada/pull/216
```

Comment:
if you want to copy/paste the github auto relase you alos need to da assed that transform those url in proper links `[#N](https://github.com/scikit-adaptation/skada/pull/N)` it is probably a god idea since it will make next releases fatser